lab/goals.md

```diff
@@ -0,0 +1,30 @@
+# OPI Lab Goals
+
+The OPI members may contribute some hardware
+to be co-managed between the member and the lab staff.
+Having hardware in the OPI lab helps
+to validate compatibility and to avoid regressions.
+
+## Usage
+
+The OPI lab should be used for:
+
+- automatic testing
+
+  * scheduled functional validation
+  * patch validation (CI)
+
+- demonstration
+
+  * allowed users might reserve their own machine
+
+The data generated in the lab shall not be used
+for competitive marketing purposes.
+
```

Comment:
I think it can also be used as development platform for OPI engineers. Think of folks who do not work for a DPU or IPU vendor, but who want to contribute and work with the devices. So this is a third use case.

Reply:
I think that if we allow other members to use hardware used for the automatic tests, then it will become very chaotic, so the vendors won't provide hardware.

Reply:
I disagree, because we can use a bare metal provisioning service to allocate resources to users as requested.

Reply:
Do you really want anybody to play with your firmware and any platform tuning while some automatic tests are running?

Reply:
Obviously I don't, which ia why the bare metal provisioning would allow a single user to use a machine at a time.